lib/componentFactory.js

```diff
@@ -241,6 +241,122 @@ module.exports = function (element) {
         inner
       );
 
+    // <price-table>
+    case this.components.priceTable:
+      var title = element.attr("title");
+      var total = element.attr("total");
+
+      if (title && element) {
+        return format(
+          '<table class="pricing-table" role="table"><tr><td><h5>%s</h5></td><td><p class="text-right"><strong>%s</strong></p></td></tr></table>',
+          title,
+          total
+        );
+      }
+
+      return format(
+        '<table class="pricing-table" role="table">%s</table>',
+        inner
+      );
+
+    // <price-header>
+    case this.components.priceHeading:
+      return format("<tr>%s</tr>", inner);
+
+    // <price-header>
+    case this.components.priceBody:
+      return format("<tr>%s</tr>", inner);
+
+    // <fa-button>
+    case this.components.faButton:
+      var expander = "";
+
+      function getColors() {
+        let colorClass = element.attr("class").split(" ");
+        let colors = {};
+        if (colorClass.includes("success")) {
+          colors.bg = "#63E6BE";
+          colors.text = "#183153";
+        } else if (colorClass.includes("alert")) {
+          colors.bg = "#FF8787";
+          colors.text = "#183153";
+        }
+
+        return colors;
+      }
+
+      let colors = getColors();
+
+      // Prepare optional target attribute for the <a> element
+      var target = "";
+      if (element.attr("target")) {
+        target = " target=" + element.attr("target");
+      }
+      // If we have the href attribute we can create an anchor for the inner of the button;
+      if (element.attr("href")) {
+        inner = format(
+          '<a %s href="%s"%s>%s</a>',
+          attrs,
+          element.attr("href"),
+          target,
+          inner
+        );
+      }
+
+      // If the button is expanded, it needs a <center> tag around the content
+      if (element.hasClass("expand") || element.hasClass("expanded")) {
+        inner = format("<center>%s</center>", inner);
+        expander = '\n<td class="expander"></td>';
+      }
+
+      // The .button class is always there, along with any others on the <button> element
+      var classes = ["button"];
+      if (element.attr("class")) {
+        classes = classes.concat(element.attr("class").split(" "));
+      }
+
+      return format(
+        `<!--[if mso]><div><v:roundrect xmlns:v="urn:schemas-microsoft-com:vml"
+    xmlns:w="urn:schemas-microsoft-com:office:word"
+    href="https://fontawesome.com/icons"
```

Comment:
Looks like some values here are hard-coded (URL, height, width, and button text especially) that we may want to have as variables?